synth.py

```diff
@@ -18,6 +18,7 @@
 
 java.common_templates(excludes=[
     'README.md',
+    'samples',
```

Comment:
```suggestion
    'samples/*',
```
synthtool uses glob format, we'll need to add the *